lib/doctor/Doctor.ts

```diff
@@ -116,21 +120,45 @@ export class Doctor {
     const localI18nKey = `${i18nKey}.accountChecks`;
     try {
       await accessTokenForPersonalAccessKey(this.accountId!, true);
+      const pakScopes = new Set(await scopesOnAccessToken(this.accountId!));
+      const missingScopes = (
+        await authorizedScopesForPortalAndUser(this.accountId!)
+      ).filter(
+        data => data.userAuthorized && !pakScopes.has(data.scopeGroup.name)
+      );
+
       this.diagnosis?.addCLIConfigSection({
         type: 'success',
         message: i18n(`${localI18nKey}.active`),
       });
-      this.diagnosis?.addCLIConfigSection({
-        type: 'success',
-        message: i18n(`${localI18nKey}.pak.valid`, {
-          link: uiLink(
-            i18n(`${localI18nKey}.pak.viewScopes`),
-            `${getHubSpotWebsiteOrigin(
-              this.diagnosticInfoBuilder?.env || 'PROD'
-            )}/personal-access-key/${this.diagnosticInfo?.account.accountId}`
+
+      const linkToPakUI = uiLink(
+        i18n(`${localI18nKey}.pak.viewScopes`),
+        `${getHubSpotWebsiteOrigin(
+          this.diagnosticInfoBuilder?.env || 'PROD'
```

Comment:
This can be simplified to `getHubSpotWebsiteOrigin(this.diagnosticInfoBuilder?.env)` b/c the util will default to the prod domain if no env is passed in. [util](https://github.com/HubSpot/hubspot-local-dev-lib/blob/main/lib/urls.ts#L8)

Reply:
is that true? the [util](https://github.com/HubSpot/hubspot-local-dev-lib/blob/e67718e153e33fb1929fecfe00e7892914c45277/lib/urls.ts#L8) it's calling requires it to be defined, so we'd either need to cast it with `!` or do that fallback. any preference?

Reply:
Ah shoot, you're right. This looks like a typing mistake over in `getHubSpotWebsiteOrigin`. The env arg should be typed more similarly to the way that our CLI_Account.env is typed, which allows undefined b/c we always fall back to prod behavior.

You can keep this as is 👍